attestation-agent/lib/src/token.rs

```diff
@@ -16,7 +16,15 @@ struct Message {
 
 pub(crate) async fn get_kbs_token() -> Result<Vec<u8>> {
     let evidence_provider = Box::new(NativeEvidenceProvider::new()?);
-    let kbs_host_addr = get_kbs_host_from_cmdline().await?;
+
+    // Check for /peerpod/daemon.json to see if we are in a peer pod
+    // If so we need to read from the agent-config file, not /proc/cmdline
+    let kbs_host_addr = if Path::new("/peerpod/daemon.json").exists() {
```

Comment:
It is encouraged to add a `const`. s.t.
```rust
const PEER_POD_CONFIG_PATH: &str = "/peerpod/daemon.json";
...
let kbs_host_addr = if Path::new(PEER_POD_CONFIG_PATH).exists() {
```

Reply:
I might use a match here instead of an if, but it's not a big deal.